ibis/common/geospatial.py

```diff
@@ -19,22 +19,38 @@ def _format_point_value(value: Iterable) -> str:
     return ' '.join(str(v) for v in value)
 
 
-def _format_linestring_value(value: Iterable) -> str:
+def _format_linestring_value(value: Iterable, nested=False) -> str:
     """Convert a iterable with a linestring to text."""
-    return ', '.join(_format_point_value(point) for point in value)
+    template = '({})' if nested else '{}'
```

Comment:
Is there any reason not to always add parentheses here? Does adding them change the semantics of the expression?

Reply:
I just tested extra parenthesis using this https://rdrr.io/cran/wellknown/man/lint.html
and it seems the extra parenthesis will fail.